frontend/src/models/Federation.model.ts

```diff
@@ -109,10 +109,11 @@ export class Federation {
 
   updateBook = async (): Promise<void> => {
     this.loading = true;
+    this.book = [];
     this.triggerHook('onCoordinatorUpdate');
     this.exchange.loadingCoordinators = Object.keys(this.coordinators).length;
     for (const coor of Object.values(this.coordinators)) {
-      await coor.updateBook(() => {
+      coor.updateBook(() => {
```

Comment:
These 2 awaits were waiting for a coordinator to load to fetch the next one, increasing the waiting time for the user

Reply:
You can notice books load time now decreased a lot